Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -0,0 +1,235 @@
+$fabricInstallUtilities = ".\Install-Identity-Utilities.psm1"
+Import-Module -Name $fabricInstallUtilities -Force
+
+# Import AzureAD
+$minVersion = [System.Version]::new(2, 0, 2 , 4)
+$azureAD = Get-Childitem -Path ./**/AzureAD.psm1 -Recurse
+if ($azureAD.length -eq 0) {
+    $installed = Get-Module -Name AzureAD
+    if ($null -eq $installed) {
+        $installed = Get-InstalledModule -Name AzureAD
+    }
+
+    if (($null -eq $installed) -or ($installed.Version.CompareTo($minVersion) -lt 0)) {
+        Write-Host "Installing AzureAD from Powershell Gallery"
+        Install-Module AzureAD -Scope CurrentUser -MinimumVersion $minVersion -Force
+        Import-Module AzureAD -Force
+    }
+}
+else {
+    Write-Host "Installing AzureAD at $($azureAD.FullName)"
+    Import-Module -Name $azureAD.FullName
+}
+
+function Get-GraphApiReadPermissions() {
+    $aad = (Get-AzureADServicePrincipal | Where-Object {$_.ServicePrincipalNames.Contains("https://graph.microsoft.com")})[0]
```

Comment:
This is kind of awkward, but didn't immediately see a good way to do this, can revisit